src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,20 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        } else {
+            String binaryNumber = "";
+            while (value >= 1) {
+                if (value % 2 != 0) {
+                    binaryNumber += "1";
+                } else {
+                    binaryNumber += "0";
+                }
+                value /= 2;
+            }
+            StringBuilder builder = new StringBuilder(binaryNumber).reverse();
+            return builder.toString();
```

Comment:
```suggestion
            return new StringBuilder(binaryNumber).reverse().toString();
```

Reply:
Sorry for so many commits.. I didn't understood all my faults from the first time))